packages/siopv2-oid4vp-op-auth/src/session/OID4VP.ts

```diff
@@ -159,16 +159,16 @@ export class OID4VP {
       format: opts?.restrictToFormats ?? selectedVerifiableCredentials.definition.definition.format,
       skipDidResolution: opts?.skipDidResolution ?? false,
     })
-
+    const identifier: ManagedIdentifierResult = await this.session.context.agent.identifierManagedGet(idOpts)
     const presentationResult = await this.getPresentationExchange({
       verifiableCredentials: vcs.credentials,
       allDIDs: this.allDIDs,
       hasher: opts?.hasher,
     }).createVerifiablePresentation(vcs.definition.definition, vcs.credentials, signCallback, {
       proofOptions,
       // fixme: Update to newer siop-vp to not require dids here.
-
-      holderDID: isManagedIdentifierDidOpts(idOpts) ? (await this.session.context.agent.identifierManagedGetByDid(idOpts)).did : undefined,
+      // holderDID: isManagedIdentifierDidOpts(idOpts) ? (await this.session.context.agent.identifierManagedGetByDid(idOpts)).did : undefined,
+      holderDID: identifier.kid,
```

Comment:
That should not be the fix for the above code. The code before was explicitly only setting the holderDID when it actually was a DID. The fixme was about the fact that we seem to rely on DIDs. So without having looked at it the 'holderDID' property should probably change.

Reply:
`holderDID` here is a term that is used in the pex only and as far as I understand doesn't have to be a did. it can be easily refactored to be just `holder`. therefore I don't think we should enforce it being a did. so in the OP side, I think, we can use the `kid` of a managed identifier.